backend/core/logging.py

```diff
@@ -0,0 +1,62 @@
+import sys
+from typing import Literal, TypeAlias
+
+import loguru
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+logger = loguru.logger
+
+LogLevel: TypeAlias = (
```

Comment:
Литералы чаще всего перечисляют так, немного компактнее)

```suggestion
LogLevel: TypeAlias = Literal['TRACE', 'DEBUG', 'INFO', ...]
```